lib/client.js

```diff
@@ -597,6 +597,21 @@ MqttClient.prototype.end = function (force, cb) {
 }
 
 /**
+ * removeOutgoingStore - remove a message in outgoing store
+ *
+ * @param {Number} mid - messageId to remove message
+ * @returns {MqttClient} this - for chaining
+ * @api public
+ *
+ * @example client.removeOutgoingStore(client.getLastMessageId());
+ */
+MqttClient.prototype.removeOutgoingStore = function (mid) {
```

Comment:
the name could be a bit more meaningful, something like `removeOutgoingMessage`.